components/raftstore/src/store/util.rs

```diff
@@ -871,7 +871,7 @@ impl RegionReadProgressRegistry {
             .lock()
             .unwrap()
             .get(region_id)
-            .map(|rp| rp.core.lock().unwrap().applied_index)
```

Comment:
Why change this?

Reply:
This method is just used in UT if we need to get tracked_index.
The tracked_index is [`read_state.idx`](https://github.com/tikv/tikv/blob/1a9446f334c2d29417f656fa8ef4ec6a1dda95f7/components/raftstore/src/store/util.rs#L1260) but not `applied_index`, so I change this.